dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java

```diff
@@ -134,7 +134,7 @@ private Date buildDate(int month, int year) {
         return c.getTime();
     }
 
-    private Date addDates(Date d, int days) {
+    private static Date addDates(Date d, int days) {
```

Comment:
Better to leave it as non-static. To make it static private it does not really make any difference in my opinion.